scipy/stats/_morestats.py

```diff
@@ -3957,14 +3960,17 @@ def directionalmean(samples, *, axis=0, normalize=True):
     Examples
     --------
     >>> from scipy.stats import directionalmean
-    >>> data = np.array([[0.8660254, 0.5, 0.], [0.8660254, -0.5, 0.]])
+    >>> data = np.array([[3, 4],    # first observation, 2D vector space
+    ...                  [6, -8]])  # second observation
     >>> directionalmean(data)
-    array([1., 0., 0.])
+    array([1., 0.])
 
-    The *regular* sample mean in contrast does not lie on the unit sphere.
+    In contrast, the regular sample mean of the vectors would be influenced
+    by the magnitude of each observation. Furthermore, the result would not be
+    a unit vector.
 
     >>> data.mean(axis=0)
-    array([0.8660254, 0., 0.])
+    array([4.5, -2.])
 
```

Comment:
I still think the example of geographical locations would be helpful for the average user, so I added it again.

```suggestion
An exemplary use case for `directionalmean` is to find a *meaningful* center for a set of observations on a sphere, e.g. geographical locations.

>>> data = np.array([[0.8660254, 0.5, 0.],
...                  [0.8660254, -0.5, 0.]])
>>> directionalmean(data)
array([1., 0., 0.])

The regular sample mean on the other hand yields a result which does not lie on the surface of the sphere.

>>> data.mean(axis=0)
array([0.8660254, 0., 0.])
```